src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -712,6 +697,56 @@ public Properties getConsumerProperties() {
     return consumerProps;
   }
 
+  public Properties getAdminProperties() {
+    Properties consumerProps = new Properties();
+    //copy cover the properties with prefixes "client." and  "admin."
+    addPropertiesWithPrefix("client.", consumerProps);
+    addPropertiesWithPrefix("admin.", consumerProps);
+    return consumerProps;
+  }
+
+  public String bootstrapBrokers() {
```

Comment:
not suggesting this be moved, i think it's a good enough factoring all things considered.